scripts/setup_e2e_docker.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const {spawn, exec} = require('child_process');
+
+const axios = require('axios');
+
+const mmctlPath = process.env.MMCTL_PATH || 'mmctl';
+
+const ping = setInterval(async () => {
+    try {
+        const pingRequest = await axios.get('http://localhost:8065/api/v4/system/ping');
+        if (pingRequest.status === 200) {
+            const addUserRequest = await axios.post(
+                'http://localhost:8065/api/v4/users',
+                {
+                    email: 'test@test.com',
+                    username: 'admin1',
+                    password: 'Sys@dmin123',
```

Comment:
would it be possible for the CI to setup these as config variables instead of hardcoding them?